src/background/sessionBackground.ts

```diff
@@ -0,0 +1,48 @@
+import Background from '.';
+import { MESSAGE_TYPE, RESPONSE_TYPE } from '../constants';
+
+export default class SessionBackground {
+  private static SESSION_LOGOUT_INTERVAL_MS: number = 600000;
+
+  public sessionTimeout?: number = undefined;
+
+  private bg: Background;
+
+  constructor(bg: Background) {
```

Comment:
you need to call `this.bg.onInitFinished('session');` at the end of the constructor to alert the main `Background` class that the the Session's init is finished and it triggers the boolean switch.

Reply:
Ya there's a bug with getting it to load the main screen after the BG is fully inited. But should be added anyways so it works when I fix the bug.